touch-events/touch-globaleventhandler-interface.html

```diff
@@ -3,7 +3,11 @@
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
 <script>
+function supportsTouchEvents() {
+  assert_implements_optional('ontouchstart' in document, "'expose legacy touch event APIs' is false");
```

Comment:
Nit: I don't know what `assert_implements_optional` logs in each case, but this still asserts `true`.  Any chance the word "false" here could be misleading?  What if we output a string similar to Line 15 below?

Reply:
Removed `is false`.